third_party/abseil-cpp/absl/synchronization/BUILD.gn

```diff
@@ -42,6 +42,7 @@ absl_test("kernel_timeout_internal_test") {
 }
 
 absl_source_set("synchronization") {
+  if (!use_cobalt_customizations) {
```

Comment:
A comment describing why these blocks are disabled for Cobalt would be helpful

Reply:
Done